<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>261</width>
      <height>308</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Pro_name">
       <property name="text">
        <string>ProJect Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Pro_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="CB_clock">
       <property name="text">
        <string>CLOCK</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="CB_segment">
       <property name="text">
        <string>8-Segment√ó8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="CB_LED">
       <property name="text">
        <string>LED</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="CB_switch">
       <property name="text">
        <string>Switch</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="CB_button">
       <property name="text">
        <string>Button</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="CB_ROM">
       <property name="text">
        <string>Accelerometer</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="CB_VGA">
       <property name="text">
        <string>VGA</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="CB_SDRAM">
       <property name="text">
        <string>SDRAM</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="CB_arduion">
       <property name="text">
        <string>Arduion Header</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_GPIO">
       <property name="text">
        <string>GPIO Header</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QComboBox" name="CBB_GPIO">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>359</y>
      <width>671</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="PB_save">
         <property name="text">
          <string>Save Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_load">
         <property name="text">
          <string>Load Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_generate">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PB_exit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="GV_fpga">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="GV_logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/img/logo.png);</string>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
    <property name="viewportUpdateMode">
     <enum>QGraphicsView::SmartViewportUpdate</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
